openedx_tagging/core/tagging/rest_api/v1/serializers.py

```diff
@@ -176,6 +176,7 @@ def to_representation(self, instance: list[ObjectTag]) -> dict:
         """
         Convert this list of ObjectTags to the serialized dictionary, grouped by Taxonomy
         """
+        ObjectTagViewMinimalSerializer = self.context["view"].minimal_serilizer_class
```

Comment:
I do it this way so that that serializer can be overridden. In my other [PR](https://github.com/openedx/edx-platform/pull/35596/files#diff-68d305b73e25deae855bf8da30edd62fcadb6cc69c61288139359ff2988ffec7R105) I override that serializer to override `get_can_delete_objecttag` and prevent the user from deleting copied tags.

Reply:
Would be good to mention this in a comment: "# Allows consumers like edx-platform to override this"

Reply:
Updated: https://github.com/openedx/openedx-learning/pull/236/commits/7289b2231ae0f7c0721327552d59501dbd8b2461